init.c

```diff
@@ -1184,6 +1184,67 @@ static enum CommandResult parse_group(struct Buffer *buf, struct Buffer *s,
   return MUTT_CMD_WARNING;
 }
 
+/**
+ * find_config_variable - return HashElem of a config_variable
+*/
+static struct HashElem* find_config_variable(const char * name)
+{
+  return cs_get_elem(Config, name);
+}
+
+/**
+ * find_my_variable - return my_variable
+*/
+static const char* find_my_variable(const char * name)
+{
+  return myvar_get(name);
+}
+
+/**
+ * find_environment_variable - return env varible
+*/
+static const char* find_environment_variable(const char * name)
+{
+  return mutt_str_getenv(name);
+}
+
+/**
+ * is_enabled_feature - return bool if argument is an enabled feature
+*/
+static bool is_enabled_feature(const char * name)
+{
+  return feature_enabled(name);
+}
+
+/**
+ * is_function - return bool if argument is a neomutt function
+*/
+static bool is_function(const char * name)
+{
+  bool res = false;
+  for (int i = 0; i < MENU_MAX; i++)
+  {
+    const struct Binding *b = km_get_table(Menus[i].value);
+
+    for (int j = 0; b && !res && b[j].name; j++)
+      if (mutt_str_strcmp(name, b[j].name) == 0)
+        res = true;
```

Comment:
return true;

Reply:
when I change `res = true;` to `return true;` I still need res for the for condition.

Reply:
Oh, got it. I can return right in the loop. So it's not necessary for the loop to check the condition again, escape loop and after that return.